.changeset/lucky-bees-care.md

```diff
@@ -0,0 +1,29 @@
+---
+"@comet/cms-admin": minor
+"@comet/admin": minor
+---
+
+Add support for `badgeContent` prop in `MenuItemRouterLink`.
+
+**Example usage in `masterMenuData`:**
+
+```ts
+const masterMenuData = [
+    // ...
+    {
+        type: "route",
+        primary: "Some Route",
+        to: "/someRoute",
+        badgeContent: 2,
+    },
+    // ...
+];
+```
+
+**Example usage as element:**
+
+```tsx
+<MenuItemRouterLink primary="Some Route" to="/someRoute" badgeContent={2} />
+```
+
+Extend `MenuItemAnchorLink` to define a correctly styled `LinkExternal` icon if no `secondaryAction` is passed.
```

Comment:
I would put this in a second changeset as it's a completely different topic

Reply:
Even better: Two PRs (for future scenarios) 😁

Reply:
https://github.com/vivid-planet/comet/pull/2431/commits/06ad97a4ed589a249ea784bcea9379f8b8416596